### PR TITLE
docs(guides/extending): remove `resolveConfig` which is deprecated

### PR DESCRIPTION
**Pull request by qbzzt**

### CODE REVIEW

File: docs/pages/guides/extending-a-world/index.mdx [hunk 1]
```diff
@@ -780,6 +780,16 @@ We will now fix this.
        client: { public: publicClient, wallet: burnerWalletClient },
      });
 
+     /* Update counterConfig.tables so it'll work as components */
+     const resolvedTables = storeToV1(counterConfig).tables;
```
Comment (holic) [inline docs/pages/guides/extending-a-world/index.mdx:784]: This isn't meant to be used externally and will be removed very soon.

Comment (qbzzt) [reply docs/pages/guides/extending-a-world/index.mdx:784]: We'll have another mechanism to add components to the synchronization?

Comment (holic) [reply docs/pages/guides/extending-a-world/index.mdx:784]: yes, you can pass config output's tables directly into `syncToRecs`

Comment (qbzzt) [reply docs/pages/guides/extending-a-world/index.mdx:784]: I assume you mean:

```
import counterConfig from "./counter.config";
.
.
.
  const { components, latestBlock$, storedBlockLogs$, waitForTransaction } = await syncToRecs({
    world,
    config: mudConfig,
    address: networkConfig.worldAddress as Hex,
    publicClient,
    startBlock: BigInt(networkConfig.initialBlockNumber),
    tables: counterConfig.tables,
  });
```

I tried it, but it dies in the `syncToRecs` call with this error:

```
TypeError: Cannot convert undefined or null to object
    at Function.entries (<anonymous>)
    at U4 (chunk-ACLFEHT3.js?v=0ed45cff:7222:48)
    at chunk-ACLFEHT3.js?v=0ed45cff:7225:24
    at chunk-A6CZEM5O.js?v=0ed45cff:48:69
    at Array.map (<anonymous>)
    at P (chunk-A6CZEM5O.js?v=0ed45cff:48:48)
    at I4 (chunk-ACLFEHT3.js?v=0ed45cff:7225:10)
    at Q2 (chunk-ACLFEHT3.js?v=0ed45cff:7231:17)
    at Qe (chunk-ACLFEHT3.js?v=0ed45cff:7267:86)
    at setupNetwork (setupNetwork.ts:98:25)
```

Or do you mean the output not from `defineWorld` but something else?

Comment (holic) [reply docs/pages/guides/extending-a-world/index.mdx:784]: ahh this is changed in main but not yet released, just waiting for multi namespaces (almost here, prob this week)

**Comment by holic** [comment]
`syncToRecs` now takes in `tables` from the config so this is no longer relevant, so gonna close this out 

we should remove the suggested usage of `storeToV1` as it no longer exists

**Pull request closed without merging**